README.md

```diff
@@ -784,6 +784,618 @@ Sample config params:
+| `feature_derivation_window_time_unit` | str        | Time unit of the feature derivation window. One of ``datarobot.enums.AllowedTimeUnitsSAFER`` If present, time-aware joins will be used. Only applicable when dataset1_identifier is not provided.                                                                                       |
+| `feature_derivation_windows`          | List       | List of feature derivation windows settings. If present, time-aware joins will be used. Only allowed when feature_derivation_window_start, feature_derivation_window_end and feature_derivation_window_time_unit are not provided.                                                      |
+| `prediction_point_rounding`           | List[dict] | Closest value of prediction_point_rounding_time_unit to round the prediction point into the past when applying the feature deri if present.Only applicable when dataset1_identifier is not provided.                                                                                    |
+| `prediction_point_rounding_time_unit` | str        | Time unit of the prediction point rounding. One of ``datarobot.enums.AllowedTimeUnitsSAFER`` Only applicable when dataset1_identifier is not provided.                                                                                                                                  |
+
+
+For more [create-dataset-definitions-and-relationships-using-helper-functions](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/reference/data/feature_discovery.html?highlight=DatasetDefinition#create-dataset-definitions-and-relationships-using-helper-functions), see the DataRobot documentation.
+
+---
+
+#### `ComputeFeatureImpactOperator`
+
+Creates Feature Impact job in DataRobot.
+
+Returns Feature Impact job ID
+
+Parameters:
+
+| Parameter       | Type | Description           |
+|-----------------|------|-----------------------|
+| `project_id`    | str  | DataRobot project ID  |
+| `model_id`      | str  | DataRobot model ID    |
+
+For more [request_feature_impact](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html?highlight=request_feature_impact#datarobot.models.Model.request_feature_impact), see the DataRobot documentation.
+
+---
+
+#### `ComputeFeatureEffectsOperator`
+
+Submit request to compute Feature Effects for the model.
+
+Returns Feature Effects job ID
+
+Parameters:
+
+| Parameter        | Type | Description          |
+|------------------|------|----------------------|
+| `project_id`     | str  | DataRobot project ID |
+| `model_id`       | str  | DataRobot model ID   |
+
+For more [request_feature_impact](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html?highlight=request_feature_effect#datarobot.models.DatetimeModel.request_feature_effect), see the DataRobot documentation.
```

Comment:
incomplete, parameters or functions?

Reply:
what about "For more information" or "For more details"?